src/Gir/Marshal/SymbolTable.Builtin.cs

```diff
@@ -54,6 +55,20 @@ void RegisterPrimitives (bool nativeWin64)
 
 			AddType (new Primitive ("va_list", "...", ""));
 
+			var alias = new Alias {
```

Comment:
This is kind of a hack. It seems like GType is kind of a primitive since in GObject GType is used without a prefix even though the definition is in GLib.